### PR TITLE
face position changed

### PR DESCRIPTION
**Pull request by lejladzanko**
I have added display:flex to body and changed its position.

### CODE REVIEW

**Comment by elialcodes** [comment]
Great job!

**Comment by psanz22** [comment]
Yeah!!!

File: main.css [hunk 1]
```diff
@@ -12,4 +12,12 @@
     position: absolute;
     top: 50%;
     transform: translate(-50%, -60%) rotate(90deg);
+    display: flex;
```
Comment (elialcodes) [inline main.css]: I think you should delete some lines of code

File: main.css [hunk 2]
```diff
@@ -12,4 +12,12 @@
...
+    justify-content: center;
```
Comment (psanz22) [inline main.css]: I think you should delete the justify-content

**Review by psanz22 — CHANGES_REQUESTED** [review]

**Review by elialcodes — COMMENTED** [review]
Please, look at my comments

**Review by lejladzanko — COMMENTED** [review]
I have changed it, thank you!

**Review by psanz22 — APPROVED** [review]
Very good

**Pull request merged**